vault.js

```diff
@@ -1,4 +1,21 @@
 'use strict';
 module.exports = function() {
-  
+    var secretKey = 'a thing'
+    var secretValue = 'two things'
```

Comment:
1.) Use `const` here. 
2.) Can you think of something we can use so that this could handle more than one key-value pair?